app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer.go

```diff
@@ -520,22 +530,22 @@ func (l *LoadBalancerSyncer) ingToNodePorts(ing *v1beta1.Ingress, client kubecli
 	var knownPorts []ingressbe.ServicePort
 	defaultBackend := ing.Spec.Backend
 	if defaultBackend != nil {
-		port, err := l.getServiceNodePort(*defaultBackend, ing.Namespace, client)
+		port, err := kubeutils.GetServiceNodePort(*defaultBackend, ing.Namespace, client)
 		if err != nil {
-			glog.Errorf("%v", err)
+			fmt.Println("error getting service nodeport:", err, "Ignoring.")
```

Comment:
Add ". " before ignoring as you do for the error below?

Reply:
done.